repro2fhirunittest.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from reproschema_to_fhir.fhir import add_options, Config, generate_code_system, generate_value_set
+
+class TestReprotoFhir(unittest.TestCase):
```

Comment:
I prefer pytest to unittest. It's mostly compatible, except you write your tests as individual functions, rather than methods of a class. You won't see any differences if you remove the class, and rename your tests from `testcase_...` to `test_...`.